### DISCUSSION TITLE
Vue routing hard reload?

### DISCUSSION THREAD

**Discussion by max3xyz** [statamic/cms#8322] [Troubleshooting] [answered]
Hi guys.
I just installed a fresh installation with version 4. I started the whole thing with `php artisan serve` locally. Now I noticed in CP that every time I switch to a new menu link in the navigation or even when I save an entry, that every time the complete page is hard reloaded. I thought Statamic uses Vue for the frontend. Am I doing something wrong, is this a bug or is this by design? 

Thanks for your work 
Max

**Comment by jackmcdade** [comment, marked as answer]
This is by design. The control panel is not an SPA. Some things happen without a refresh, but most atomic actions, like saving etc, have a refresh. On Jun 17, 2023, at 12:17 PM, Max Hanauer ***@***.***> wrote:﻿
Hi guys.
I just installed a fresh installation with version 4. I started the whole thing with php artisan serve locally. Now I noticed in CP that every time I switch to a new menu link in the navigation or even when I save an entry, that every time the complete page is hard reloaded. I thought Statamic uses Vue for the frontend. Am I doing something wrong, is this a bug or is this by design?
Thanks for your work
Max

**Comment by max3xyz** [comment]
Thanks for the quick reply 😊. Ah, I see. I'm just starting over with Statamic, so sorry for the stupid question. I'm so used to SPAs in control panels by now that the hard reload felt weird right away.
